Project EFT/Data Classes/Admin.cs

```diff
@@ -28,5 +28,10 @@ public static Admin Decode()
         {
             return new Admin();
         }
+
+        public bool IsEqual(Admin otherAdmin)
+        {
+            return (this.Username.Equals(otherAdmin.Username) && this.Password.Equals(otherAdmin.Password) && this.Email.Equals(otherAdmin.Email) && this.Id == otherAdmin.Id);
+        }
```

Comment:
no check for submission equality?

Reply:
The test this is used for, ```testAdminLogin()``` returns an Admin with no submission list.  I have added an ```IsEqualWithSubList(Admin otherAdmin)``` in my next commit now, but it is currently not used.  Perhaps it would be used in some form of insert/delete?